### DISCUSSION TITLE
Add tags to multiple videos

### DISCUSSION THREAD

**Discussion by lamyergeier** [stashapp/stash#4737] [Q&A] [answered]
In the following screenshot, I would like to add a common tag to all the following videos, is it possible to select all at once and add the tag?

![image](https://github.com/stashapp/stash/assets/42092626/cf5797a0-6e4c-40f0-ae24-820af6de945d)

**Comment by AdultSun** [comment, marked as answer]
Yep, just click the overflow menu (3 dots), "select all", "edit" icon (pencil), then select whatever tag you need in the mass edit dialog that pops up.

![image](https://github.com/stashapp/stash/assets/64123896/902b2aba-9b46-43ab-956e-23f83df7feb7)

![image](https://github.com/stashapp/stash/assets/64123896/8f5c4806-64f8-4d7b-b68c-208776a179c3)
